### ISSUE TITLE
Semicolons after break statement unrecognized

### ISSUE THREAD

**Issue by Hexcede** [fstirlitz/luaparse#108]
`break` statements like the following are unrecognized by the lexer and result in an error:
```lua
for _, value in ipairs(tab) do
    if something then
        break; -- Here
    end
end
```

This is using lua version `5.1`, I haven't tried any other versions.

This only seems to happen with `break` statements. Currently I am trying to work with automatically generated code which is placing semicolons after `break` statements. I unfortunately have to remove any breaks with semicolons otherwise I can't generate an AST despite it being valid.

**Comment by fstirlitz** [comment]
Confirmed. As a workaround, you can try parsing in 5.2 mode, since it has relaxed syntax constraints `break` statements.

The fix is easy, though, and will be committed soon.

**Comment by fstirlitz** [comment]
Resolved by d61c6fe1cd2757bc210898bc9b263ffe4adbd060, thank you.

**Comment by Hexcede** [comment]
> Resolved by [d61c6fe](https://github.com/fstirlitz/luaparse/commit/d61c6fe1cd2757bc210898bc9b263ffe4adbd060), thank you.
Awesome, no problem! Thanks for such a quick resolution.
